src/main/scala/ai/privado/tagger/sink/SinkArgumentUtility.scala

```diff
@@ -0,0 +1,291 @@
+package ai.privado.tagger.sink
```

Comment:
Any reason to move the above collection Taggers outside of the collection package?

Reply:
In taggers we have
- sinks
- collections
but initiallly direct `sink.collection` package created which not allowing me to create another folder package.